gooddata-sdk/type-requirements.txt

```diff
@@ -2,3 +2,6 @@ mypy==0.910
 pydantic==1.8.2
 types-python-dateutil >= 2.5.3
 types-pyyaml~=6.0.0
+pyyaml==6.0
```

Comment:
Why is necessary to install pyyaml for mypy tests? `types-pyyaml` should be enough?

Reply:
You are right. Copy paste error, thanks for spotting it.